GoogleForCreators/web-stories-wp
# Dashboard: Adjust Filter Dropdown Width

## Context



## Summary
Allow for multiple filters to be displayed on the Dashboard without overflowing.  Also allow for filter labels to expand the size of the filter, and drop down section based on the filter name length, option name length and the search placeholder length.


## Relevant Technical Choices
Used arbitrary max width for filters, options, and placeholder text to keep the filters reasonable size.



## To-do



## User-facing changes


Before:
![Before Filters](https://user-images.githubusercontent.com/14057928/175321268-e7f3d9a8-a6be-4b9c-9e12-877c90543df0.png)

After:
![After Filters](https://user-images.githubusercontent.com/14057928/175321289-beaeade9-55d5-4241-b097-756f3c8fe4e5.png)


## Testing Instructions






This PR can be tested by following these steps:

1. Go to Dashboard
2. Make sure you have Multiple taxonomies 
- I used 'CPT UI' plugin to make a few filters with long text and then used the Story Editor to add taxonomy options
- Make sure to set 'Hierarchical' to 'true' for the filter you create with 'CPT UI'
3. Click on the 'Switch to list view' icon

On Staging:
- Notice filters can overflow the visible space, make the user have to scroll horizontally to  see other filters
- Notice there filters are not on the list view

On Branch:
- Notice filters widths vary, filters do not overflow visible space they stack in a grid formation.
- Notice the filters are on the list view


## Reviews

### Does this PR have a security-related impact?
No


### Does this PR change what data or activity we track or use?
No


### Does this PR have a legal-related impact?
No


## Checklist




---



Fixes #11665
